### PR TITLE
Unlink Action added. Actions with issues events refactoring.

### PR DESCRIPTION
**Pull request by ArtsiomYeliseyenka**

### CODE REVIEW

File: src/main/resources/public/js/src/launches/logLevel/LogItemInfoTabs/LogItemInfoActivity.js [hunk 1]
```diff
@@ -135,7 +136,7 @@ define(function (require) {
             isHideNameCell: {
                 deps: ['actionType'],
                 get: function (actionType) {
-                    return actionType === 'analyze_item' || actionType === 'attach_issue_aa';
+                    return actionType === 'analyze_item' || actionType === 'attach_load_aa';
```
Comment (DiscoElevator) [inline src/main/resources/public/js/src/launches/logLevel/LogItemInfoTabs/LogItemInfoActivity.js]: `load_issue_aa`?

File: src/main/resources/public/js/src/launches/multipleActions/unlinkIssueAction.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                        var issue = item.getIssue();
+                        issue.externalSystemIssues = [];
+                        data.issues.push({
+                            test_item_id: item.get('id'),
+                            issue: issue
+                        });
+                    }, this);
+
+                    return CoreService.updateDefect(data).done(function () {
+                        Util.ajaxSuccessMessenger(message);
```
Comment (DiscoElevator) [inline src/main/resources/public/js/src/launches/multipleActions/unlinkIssueAction.js]: I don't see where `message` receives any string. At this moment its' value is `undefined`, isn't it?

File: src/main/resources/public/js/src/projectEvents/projectEventsItemView.js [hunk 3]
```diff
@@ -67,7 +67,11 @@ define(function (require) {
                         var $valueView;
                         var oldLinks = [];
                         var newLinks = [];
-                        if (this.view.model.get('actionType') === 'post_issue' || this.view.model.get('actionType') === 'attach_issue') {
+                        if (
+                            this.view.model.get('actionType') === 'post_issue'
```
Comment (DiscoElevator) [inline src/main/resources/public/js/src/projectEvents/projectEventsItemView.js]: `this.view.model.get('actionType')` - I think it's time to make a variable for this

File: src/main/resources/public/js/src/launches/multipleActions/unlinkIssueAction.js (hunk 2 above)
Comment (ArtsiomYeliseyenka) [reply src/main/resources/public/js/src/launches/multipleActions/unlinkIssueAction.js]: Fixed

File: src/main/resources/public/js/src/launches/logLevel/LogItemInfoTabs/LogItemInfoActivity.js (hunk 1 above)
Comment (ArtsiomYeliseyenka) [reply src/main/resources/public/js/src/launches/logLevel/LogItemInfoTabs/LogItemInfoActivity.js]: fixed

File: src/main/resources/public/js/src/projectEvents/projectEventsItemView.js (hunk 3 above)
Comment (ArtsiomYeliseyenka) [reply src/main/resources/public/js/src/projectEvents/projectEventsItemView.js]: Fixed

**Pull request merged**